src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -61,6 +71,9 @@ public void init(@NotNull ConnectionManager connectionManager, @NotNull String e
         this.connectionManager.set(connectionManager);
         this.subscribeTarget.set(subscribeTarget);
         this.exchangeName.set(exchangeName);
+
+        livenessMonitor = livenessArbiter.register("rabbit_subscriber_liveness");
```

Comment:
We should register monitor with subscribe target name, because we can create much subscribers for different queues.